### PR TITLE
 Adding set-up steps to readme

### LINKED ISSUE #1: 🗞 Add Setup section to README.md file
📄 Add Setup section to README.md file
---

To allow easy setup of the project, create a **Setup** section in the `README.md` file and add steps on how locally allow the setup of the project.

The _Setup_ Section must include the following -

* Create a Fork
* `git clone` the project
* Check for Prerequisites for the project
* Commands for Initiating and running the project

### PR DESCRIPTION
**Pull request by Snehil0603**
Fixes #1 -:Adding set-up steps to readme

### CODE REVIEW

**Comment by PrerakMathur20** [comment]
Please add a detailed description of the Work you've made. Moreover, follow the widely accepted open source practices by reserving the PR Title for the work you've done and linking the Issue in the `Fixes #<Issue Number>` format in the description box.

Also, the _Setup_ section can look better and more detailed, if we take proper benefit of the Markdown(`.md`) file format.
Take a look [Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

**Comment by PrerakMathur20** [comment]
@Snehil0603 

This Setup is not quite up to the mark. Kindly look around other projects and Repositories and try to match those `README.md` file's _Setup_ section.

**Comment by PrerakMathur20** [comment]
I believe we don't need the contributing section in the Setup. 
The _Setup_ Section should only convey the directions and Methods on how to locally run the proect.

**Comment by PrerakMathur20** [comment]
@Snehil0603 Any Updates?

**Comment by Snehil0603** [comment]
@PrerakMathur20 sir I really could not get what more changes I can make to suit the repo as my commit before did not have contributing steps but it was not accepted .

File: README.md [hunk 1]
```diff
@@ -3,3 +3,17 @@
...
+* Clone the project using git clone `https://github.com/@username/geet-hub-frontend`
+* Open the folder in your favorite editor and cd geet-hub-frontend/
+* To install all dependencies run -: `npm i`
+* On the command line run `npm run build` -:It creates a build directory with a production build of your app.
+* To start the project on localhost run `npm run start`-: It runs the node script that is listed under start in the package.json. 
+* Go to `http://localhost:3000/` to see the application running
+* Create a feature branch and checkout to it `git checkout -b newFeature`
+* Stage and commit your changes using `git add .` and  `git commit -m "some message"` respectively.
+* Push to remote `git push -u origin newFeature`
+* Open a pull request and link it to a chosen issue by adding Fix #[Issue Number] to the title.
```
Comment (PrerakMathur20) [inline README.md:18]: These steps are unnecessary under the _Setup_. 
They should be a part of the _Contributing_ title, which is irrelevant as of now. So kindly remove these.

**Comment by PrerakMathur20** [comment]
LGTM!

**Pull request merged**